KakaoImageSearch/Api/KakaoImageSearchApi.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  KakaoApi.swift
+//  KakaoImageSearch
+//
+//  Created by 강현석 on 2021/02/01.
+//
+
+import Alamofire
+
+class KakaoImageSearchApi {
```

Comment:
- P3: 이 클래스가 어딘가에 상속되어지지 않는다면 `final` 키워드를 선언하는 것이 좋습니다 :)
- P4: 변수명을 지을 때 약어가 대문자로 시작한다면 약어 모두 대문자로 표기해주는 것이 더 스위프트스럽습니다

```suggestion
final class KakaoImageSearchAPI {
```

Reply:
꿀팁 감사합니다 👍